webapp/packages/core-blocks/src/Snackbars/SnackbarCommonStyles.ts

```diff
@@ -43,25 +42,53 @@ export const SNACKBAR_COMMON_STYLES = composes(
       }
     }
 
-    notification-header {
+    notification-body {
       display: flex;
+      flex-direction: column;
+      flex: 1 0 0;
+      & body-text-block {
+        margin-top: 8px;
+        & text-block-title {
+          font-size: 20px;
+          line-height: 25px;
+          font-weight: 700;
+          margin: 0;
+          padding: 0;
+          margin-bottom: 8px;
+        }
+        & message {
+          font-size: 16px;
+          display: flex;
+          align-items: center;
+          opacity: 0.8;
+          flex: 1;
+          overflow: auto;
+          max-height: 200px;
+          padding-right: 24px;
+          margin-bottom: 8px;
+          word-break: break-word;
+        }
+      }
     }
-
-    message {
-      flex: 1;
-      overflow: auto;
-      max-height: 200px;
-      padding-right: 24px;
-      word-break: break-word;
+    
+    notification-footer {
+      display: flex;
+      align-items: flex-end;
+      justify-content: space-between;
+      & footer-time {
+        font-size: 12px;
+        color: #8b8b8b;
+      }
```

Comment:
use `composes: theme-typography--caption from global;` here and remove color constant, you can try opacity, but avoid to use not themed color constants in component styles